docs/config.md

```diff
@@ -715,6 +715,16 @@ The following settings are available:
 `google.enableRequesterPaysBuckets`
 : When `true` uses the given Google Cloud project ID as the billing project for storage access. This is required when accessing data from *requester pays enabled* buckets. See [Requester Pays on Google Cloud Storage documentation](https://cloud.google.com/storage/docs/requester-pays) (default: `false`).
 
+`google.httpConnectTimeout`
+: :::{versionadded} 23.06.0-edge
+  :::
+: Defines the HTTP connection timeout for Cloud Storage API requests (default: `'20s'`).
+
+`google.httpReadTimeout`
+: :::{versionadded} 23.06.0-edge
+  :::
+: Defines the HTTP read timeout for Cloud Storage API requests (default: `'20s'`).
```

Comment:
Are these the stock defaults? it may be better to increase them

Reply:
They are. and yes they should probably be higher. Not sure what the AWS defaults are but I bet they are higher

Reply:
Okay I increased the defaults to 60 seconds